server/pulp/server/db/model/workers.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from mongoengine import Document, StringField, DateTimeField
+
+from pulp.server.db.model.base import CriteriaQuerySet
+
+
+_logger = logging.getLogger(__name__)
+
+
+class Worker(Document):
+    """
+    Represents a worker.
+
+    This inherits from mongoengine.Document and defines the schema for the documents
+    in the worker collection.
+
+    :ivar worker_name:    worker name, in the form of "worker_type@hostname"
+    :type worker_name:    basestring
+    :ivar last_heartbeat:  ISO8601 representation of the time the task started executing
+    :type last_heartbeat:  basestring
+    """
+    name = StringField(primary_key=True)
+    last_heartbeat = DateTimeField()
```

Comment:
correct, this is not an iso8601 string but a "regular" datetime.datetime. I made a copy/paste error in the docstring above.